README.md

```diff
@@ -13,48 +13,6 @@ Work includes:
-
-	A [“dynamic modules” proposal](https://github.com/nodejs/modules/issues/252) was created and presented to TC39, the standards body responsible for the ECMAScript Modules spec. This was a proposal to allow new export names to be discovered during the execution phase, which happens after linking. This made named exports from CommonJS work, but had the caveat that `export *` would not be supported for CommonJS sources. Without a solution for this edge case, TC39 deemed the proposal inadequate. There are currently no other proposals in development to try to surmount these obstacles.
-
-	The approach taken by the [`standard-things/esm`](https://github.com/standard-things/esm) package is to transpile ES module sources into CommonJS on the fly, and have Node.js evaluate the CommonJS sources. This is not considered a viable option for Node.js because it means that the actual source code being executed by the V8 JavaScript engine is not the code that the user wrote.
-
-* `require` of ES modules: Support using `require` (in addition to `import()`) to load ES modules into a CommonJS context.
-	- Discussion: https://github.com/nodejs/modules/issues/299.
-	- PR: https://github.com/nodejs/node/pull/30891.
-	- **Status**: Blocked due to various issues discussed in https://github.com/nodejs/modules/issues/454.
-
-### Not Planned
-
-* Limited module type detection.
-  - PR: https://github.com/nodejs/ecmascript-modules/pull/69.
-  - Upstream PR: https://github.com/nodejs/node/pull/27808.
-  - **Status**: The group’s current thinking is to encourage explicit source type definition via file extension or the `"type"` field. Source type detection via parsing source code is not 100% accurate and should therefore be left to userland where users can choose to accept its risks.
-
-* Provide a way to make ESM the default instead of CommonJS.
-  - Discussion: https://github.com/nodejs/modules/issues/318.
-  - Proposal: https://github.com/nodejs/modules/issues/335.
-  - **Status**: Currently one can add `--input-type=module` to `NODE_OPTIONS` to flip the default for `--input-type`; at the moment the group is deciding not to pursue providing an ability to flip the default for the `package.json` `type` field. We instead want to encourage all packages, both ESM and CommonJS, to include an explicit `type` field; leaving it out and relying on a default is something we want to discourage.
-
 ## Archives
```

Comment:
We could probably call this _Resources_ and then also link to the roadmap and previous work from there.

Reply:
Would Resources then be linked from the README? I was concerned that people would never find these documents if they weren't linked from elsewhere.

Reply:
I mean specifically to have a readme section used to link to the other sections.

Reply:
There is a separate file in the root of this repo called [`RESOURCES.md`](https://github.com/nodejs/modules/blob/master/RESOURCES.md), which has a different meaning. Having a "Resources" section in the README, plus a RESOURCES file, which are unrelated seems a bit odd to me. Thoughts?

Reply:
With this thought in mind, I've been noticing a trend of having this section named "Links" instead. I'm going to going with that if there are no objections. Also, I've made drastic improvements to the triplet roadmap concept. https://github.com/DerekNonGeneric/loader339/blob/master/doc/roadmap.md

I will be bringing all of these changes into this PR within the hour so we can continue the discussion in the main PR. I don't want this to drag on.